runtime/criusupport/criusupport.cpp

```diff
@@ -143,6 +145,8 @@ setupJNIFieldIDsAndCRIUAPI(JNIEnv *env, jclass *currentExceptionClass, IDATA *sy
 		goto done;
 	}
 
+	vm->checkpointState.isJdwpEnabled = FIND_ARG_IN_VMARGS(STARTSWITH_MATCH, "-agentlib:jdwp", NULL) >= 0;
```

Comment:
Can `MAPOPT_AGENTLIB_JDWP_EQUALS "-agentlib:jdwp="` be used here?

Reply:
I dont think `-agentlib:jdwp=` is enough, you can enable debugging support with `-Xrunjdwp` as well

Reply:
Ya looks like `-Xrunjdwp` also needs to be checked. I thought it was mapped to the first option.

Reply:
Something was broken, or following comment is obsolated
https://github.com/eclipse-openj9/openj9/blob/59bdae92b1d3026620b2601275d0b3da14690334/runtime/vm/jvminit.c#L1418

Reply:
Added a check for `Xrunjdwp` as well for setting `isJdwpEnabled`